yarn.lock

```diff
@@ -6526,7 +6526,7 @@
   dependencies:
     "@types/node" "*"
 
-"@types/node@*":
+"@types/node@*", "@types/node@12.12.21":
```

Comment:
BTW it's always worth reading your `yarn.lock` to see what actually changed.

If you see a new line for a module that's already in use elsewhere in the repo, that's a _strong_ indication that you're introducing/increasing dependency version inconsistency.